src/test_utils/messages_util.rs

```diff
@@ -78,7 +68,7 @@ impl Random for messages::connect_response::ConnectResponse {
             receiver_id: Random::generate_random(),
             receiver_fob: Random::generate_random(),
             serialised_connect_request: generate_random_vec_u8(64),
-            connect_request_signature: Random::generate_random()
+            connect_request_signature: crypto::sign::Signature([0; 64]),
```

Comment:
I see this part as WIP and we should have no invalid sigs, this should be foxed in tech debt sprint though. Testing could be better.